### DISCUSSION TITLE
JSLT + Gradle

### DISCUSSION THREAD

**Discussion by mengting1010** [schibsted/jslt#254] [General]
Hi, 

Can we also have an instruction for "how to add this as dependency using Gradle" if possible? Let me know if I am missing any doc mentioning this. Thanks a lot!

Thanks,
Mengting

**Comment by larsga** [comment]
Is it not clear from the README how to do it?

If not, does [this](https://mvnrepository.com/artifact/com.schibsted.spt.data/jslt/0.1.11) help?

**Reply by mengting1010** [reply]
Thanks for the quick reply. I plan to integrate this with our production gradle so that want to double check what I add is correct. Link is helpful. thanks!
